.github/workflows/e2e.yml

```diff
@@ -67,19 +68,25 @@ jobs:
         run: |
           chmod 0600 $PGPASSFILE
           backend/db-setup/create-db.sh
-      - name: Set up nginx reverse proxy and application-local
+      - name: Set up /etc/hosts, nginx, and application-local
         run: |
-          sudo cp ./frontend/e2e/utils/nginx/localhost.simplereport.gov /etc/nginx/sites-available
-          sudo ln -s /etc/nginx/sites-available/localhost.simplereport.gov /etc/nginx/sites-enabled/localhost.simplereport.gov
           echo "127.0.0.1 localhost.simplereport.gov" | sudo tee -a /etc/hosts
+          sudo cp ./frontend/cypress/support/nginx/localhost.simplereport.gov /etc/nginx/sites-available
+          sudo ln -s /etc/nginx/sites-available/localhost.simplereport.gov /etc/nginx/sites-enabled/localhost.simplereport.gov
+          sudo systemctl start nginx
           echo "server.servlet.session.cookie.domain: localhost.simplereport.gov" >> ./backend/src/main/resources/application-local.yaml
           echo "okta.client.org-url: http://localhost:8088" >> ./backend/src/main/resources/application-local.yaml
           echo "okta.client.token: foo" >> ./backend/src/main/resources/application-local.yaml
-          sudo systemctl start nginx
+          echo "okta.oauth2.issuer: https://hhs-prime.oktapreview.com/oauth2/default" >> ./backend/src/main/resources/application-local.yaml
+      - name: Run Wiremock
+        working-directory: ./frontend
+        run: |
+          ./cypress/support/wiremock/download-wiremock.sh
+          java -jar ./wiremock-jre8-standalone-2.29.1.jar --port 8088 --root-dir ./cypress/stubs/orgSignUp --verbose --local-response-templating &> ../wiremock-log.txt &
       - name: Run API
         env:
           OKTA_API_KEY: ${{ secrets.OKTA_API_KEY }}
-          OKTA_OAUTH2_CLIENT_ID: ${{ secrets.OKTA_OAUTH2_CLIENT_ID }}
+          OKTA_OAUTH2_CLIENT_ID: 0oa1k0163nAwfVxNW1d7
```

Comment:
I think we need to take this secret out and make sure we're using the correct TF variable here (the existing one might be for prod, we should verify that this clientId is for preview)

Reply:
This client ID is for the "local dev" Okta app, the same one that's in `application-okta-local.yml`. I need to use this specific one because it's in all the new Wiremock mappings

Reply:
Ahhh ok, gotcha. Is it possible to still make that a secret? It probably doesn't matter much given that this is only for the local dev app, but I'd like to keep it out of the repo if possible

Reply:
Actually, I think all of the Okta client IDs are already visible in the repo, see [here (okta-local)](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/resources/application-okta-local.yaml#L20) and [here (okta-prod)](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/resources/application-okta-prod.yaml#L5). Is that a potential issue? I'm not sure if the client ID needs to be secret or not :thinking:

Reply:
Hmm interesting, I didn't realize that 🤔 It's likely not an issue, @jdorothy or @rin-skylight do you have any concerns?

Reply:
I don't think this is an issue since it's the client secret that we really need to protect. Here's a forum post from Okta stating as much, FWIW: https://devforum.okta.com/t/is-clientid-for-an-app-confidential/5841

Reply:
I concur with the above. We can safely rely on Okta's built-in safeguards for this. This same methodology holds true for some of the other auth platforms I've previously used, so I think we can breathe easy.

Reply:
Sounds good! In that case we should be good to leave this as-is @nickclyde.